scripts/install

```diff
@@ -121,19 +116,34 @@ yunohost service add $app --description="Manage email discussion and e-newslette
 #=================================================
 # SSOWAT CONFIGURATION
 #=================================================
+ynh_script_progression --message="Configuring permissions..." --weight=1
+
+# Make app public if necessary
+if [ $is_public -eq 1 ]
+then
+	# Everyone can access the app.
+	# The "main" permission is automatically created before the install script.
+	ynh_permission_update --permission="main" --add="visitors"
+fi
 
-ynh_script_progression --message="Configuring SSOwat..."
+ynh_permission_create --permission="admin" --url="/admin" #--allowed=$admin
```

Comment:
```suggestion
ynh_permission_create --permission="admin" --url="/admin" #--allowed=$admin
```

Eh we should probably have an actual `--allowed` here ... except we don't have an `$admin` variable so far - because in the past we were using the old weird `protected_url` mecanism ... 

Theoretically the admin of the yunohost can add themselves to the appropriate permission in the webui, but that's not super straightforward to understand, we should really just add this new question to the manifest.json